Workbooks/Azure Active Directory/Sign-ins Overview/SigninsOverview.workbook

```diff
@@ -0,0 +1,284 @@
+{
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json",
+  "version": "Notebook/1.0",
+  "isLocked": false,
```

Comment:
you'll want `"isLocked" = true` so that the workbook is in reading mode instead of editing mode (since the advanced editor only works in edit mode, it leaves the isLocked = false here in the json it generates.)

Reply:
Right now, I want to keep it "isLocked": false, so that we can play around with the templates until we come up with the final version. Will change it to true once we are ready.